ale/base/data_naif.py

```diff
@@ -150,10 +151,22 @@ def sensor_position(self):
 
     @property
     def frame_chain(self):
-        pass
-
-    @property
-    def _sensor_orientation(self):
+        # pass
+        if not hasattr(self, '_frame_chain'):
+            frame_chain = {}
+            frame_chain['j2000'] = FrameNode(1)
+            # self._frame_chain['spacecraft'] = FrameNode(self.)
+            frame_chain['sensor'] = FrameNode(self.sensor_frame_id, 
+                                                parent=frame_chain['j2000'],
+                                                rotation='ConstantRotation')
+            frame_chain['target'] = FrameNode(self.target_frame_id, 
+                                                 parent=frame_chain['j2000'],
+                                                 rotation='TimeDependentRotation')
+            self._frame_chain = frame_chain
+        return self._frame_chain
```

Comment:
The Frame Chain is a tree object, so we can just return the root node of the tree (99.99999% of the time with SPICE this is the J2000 frame).